analyzers/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicSyntaxHelper.cs

```diff
@@ -179,6 +179,7 @@ private static bool IsOn(this ExpressionSyntax expression, SyntaxKind onKind)
             SimpleNameSyntax x => x.Identifier,
             StructureBlockSyntax x => x.StructureStatement.Identifier,
             QualifiedNameSyntax x => x.Right.Identifier,
+            VariableDeclaratorSyntax x => x.Names.First().GetIdentifier(),
```

Comment:
Initialization is not allowed when multiple variables are declared on the same line (Error ID: BC30671).

Reply:
I wouldn't do this in the helper class. The reason is that the method is called `GetIdentifier` and for VariableDeclaratorSyntax you don't have a unique name in caes like this
```vb
Dim a, b As String
```
The "initialization not null" requirement is a special case for our rule but not a general requirement.